tests/unit/pause.tcl

```diff
@@ -418,15 +418,35 @@ start_server {tags {"pause network"}} {
 
 start_cluster 1 1 {tags {"external:skip cluster pause network"}} {
     test "Test check paused info during the cluster failover in info clients" {
+        set CLUSTER_PACKET_TYPE_NONE -1
+        set CLUSTER_PACKET_TYPE_FAILOVER_AUTH_ACK 6
+
         assert_equal [s 0 paused_reason] "none"
         assert_equal [s 0 paused_actions] "none"
         assert_equal [s 0 paused_timeout_milliseconds] 0
 
+        # Let replica drop FAILOVER_AUTH_ACK so that the election won't
+        # get the enough votes and the election will time out.
+        R 1 debug drop-cluster-packet-filter $CLUSTER_PACKET_TYPE_FAILOVER_AUTH_ACK
         R 1 cluster failover
         wait_for_log_messages 0 {"*Manual failover requested by replica*"} 0 10 1000
 
+        # Failover will definitely time out, so on the primary side we will pause for
+        # `CLUSTER_MF_TIMEOUT * CLUSTER_MF_PAUSE_MULT` this long.
```

Comment:
I have an idea for CLUSTER_MF_TIMEOUT to be a configurable config, what do you think? @madolson @PingXie @zuiderkwast I think an admin could configure how long a primary would be paused in the worst case scenario, such as a failover timeout (for whatever reason, votes or a bug).

Reply:
>I have an idea for CLUSTER_MF_TIMEOUT to be a configurable config

Yeah, I think this is reasonable. We had some discussions internally because the `FAILOVER` command allows passing in a timeout, which I think is preferable to a config, but it's more difficult to do that in cluster mode. I would be ok with just a config.

Reply:
good to know, i will process it in the next PR.